### ISSUE TITLE
Add an option to disable the big youtube thumbnails and channels

### ISSUE THREAD

**Issue by Newpipeuser** [ram-on/SkyTube#891]
Please add this feature it would save data no loading the YouTube thumbnails or pictures

**Comment by drogga** [comment]
Those are nothing compared to the data you will "waste" on the videos themselves (watching or downloading them)...!
All the video's thumbnails are the same size, there is not big/large or small ones.
This is the opposite of a great idea => pointless, should be closed...
...man, there are so many, way more important things that need to be fixed\done (and lack of volunteers\contributors), and then you see something like this^...

**Comment by gzsombor** [comment]
Yes, I agree with @drogga , the thumbnails are small relatively all the html which the app downloads to collect all the required informations.
